ydb/docs/en/core/reference/kafka-api/auth.md

```diff
@@ -0,0 +1,32 @@
+# Kafka API Authentication
+
+## Enabling authentication
+
+Authentication is always enabled when using the [Kafka API in Yandex Cloud](https://yandex.cloud/docs/data-streams/kafkaapi/auth).
```

Comment:
Documentation about Yandex Cloud is published on https://yandex.cloud/ru/docs instead of here.